dbms/src/Storages/DeltaMerge/Remote/RNDeltaIndexCache.h

```diff
@@ -85,7 +85,7 @@ class RNDeltaIndexCache : private boost::noncopyable
 
     struct CacheValueWeight
     {
-        size_t operator()(const CacheValue & v) const { return v.bytes; }
+        size_t operator()(const CacheKey & /*key*/, const CacheValue & v) const { return v.bytes; }
```

Comment:
We'd better add `sizeof(CacheKey)` here cause the key is composed by plenty of members